coverage_ai/FilePreprocessor.py

```
@@ -47,3 +47,33 @@ def _contains_class_definition(self) -> bool:
+        return False
+
+    def _contains_import_statement(self) -> bool:
+        """
+        Check if the file contains a Python import statement using the ast module.
+        """
+        try:
+            with open(self.path_to_file, "r") as file:
+                content = file.read()
+            parsed_ast = ast.parse(content)
+            for node in ast.walk(parsed_ast):
+                if isinstance(node, ast.Import) or isinstance(node, ast.ImportFrom):
```

Ruff (0.8.0)

Multiple `isinstance` calls for `node`, merge into a single call

Merge `isinstance` calls for `node`

(SIM101)